cmd/minikube/cmd/start.go

```diff
@@ -68,18 +70,73 @@ func runStart(cmd *cobra.Command, args []string) {
 	kubeHost = strings.Replace(kubeHost, "tcp://", "https://", -1)
 	kubeHost = strings.Replace(kubeHost, ":2376", ":443", -1)
 	fmt.Printf("Kubernetes is available at %s.\n", kubeHost)
-	fmt.Println("Run this command to use the cluster: ")
-	fmt.Printf("kubectl config set-cluster minikube --server=%s --certificate-authority=$HOME/.minikube/apiserver.crt\n", kubeHost)
-	fmt.Println("kubectl config set-credentials minikube --client-certificate=$HOME/.minikube/apiserver.crt --client-key=$HOME/.minikube/apiserver.key")
-	fmt.Println("kubectl config set-context minikube --cluster=minikube --user=minikube")
-	fmt.Println("kubectl config use-context minikube")
+
+	// setup kubeconfig
+	name := constants.MinikubeContext
+	certAuth := constants.MakeMiniPath("apiserver.crt")
+	clientCert := constants.MakeMiniPath("apiserver.crt")
+	clientKey := constants.MakeMiniPath("apiserver.key")
+	active, err := setupKubeconfig(name, kubeHost, certAuth, clientCert, clientKey)
```

Comment:
nit: Collapse this into one line since you have the else if?
if active, err :=...; err !- nil {
...
} else if !active {
...
}